### ISSUE TITLE
Possible issue on extracting embedded subtitles from older series

### ISSUE THREAD

**Issue by Myrodistus** [morpheus65535/bazarr#2211]
**Describe the bug**
After downloading the following torrent via Sonarr (https://thepiratebay.org/description.php?id=37356902), adding "Embedded Subtitles" as the only Provider and disabling "Use Embedded Subtitles" under the "Subtitles"->"Performance / Optimization" section Bazarr is unable to extract the embedded subtitles. However, when enabling "Use Embedded Subtitles", Bazarr is able to detect that the subtitles do exist in the .mkv files.

**To Reproduce**
1. Torrent any season (or episode) of Blu-ray Supernatural releases with embedded subtitles, up until season 5 - from what I've tested.
2. Remove all other providers, other than "Embedded Subtitles"
3. Disable "Use Embedded Subtitles" under the "Subtitles"->"Performance / Optimization" - as referenced in this Reddit thread https://www.reddit.com/r/bazarr/comments/kr5ka8/extract_subtitles_from_a_tv_show_via_bazarr/

Comment of steps by [baIdissara] ^

4. No subtitles can be extracted through the "Embedded Subtitles", even though Bazarr can detect they do exist when enabling the "Use Embedded Subtitles" under the "Subtitles"->"Performance / Optimization"

**Expected behavior**
Subtitles should be extracted through the "Embedded Subtitles" provider

**Screenshots**
https://imgur.com/a/hORHmYH

**Software (please complete the following information):**
 - Bazarr: 1.2.4
 - Radarr version: N/A
 - Sonarr version: 3.0.10.1567
 - OS: Windows-10-10.0.19045-SP0

**Additional context**
I have tested other subtitle extraction software, such as VideoProc, and it can also see + extract, the subtitle files that are embedded (VideoProc is terrible, please do not even attempt to use it, I just tried for testing purposes.

Out of curiosity I tried this with a newer series, S3 E8 of The Boys, and Bazarr had no issue extracting the subtitles. And was also able to detect embedded subs when "Use Embedded Subtitles" was enabled.

I have tried multiple different torrents for earlier seasons of Supernatural, and they all had the same issue.
There's a large likelihood of me doing something in-correct, but after talking with TRaSH on the Bazarr Discord server it would be great to see if someone could replicate this - or possibly correct it if it isn't just me.

If this is just an issue with this season of the series being so old, I understand. I'm just wondering why Bazarr can see that embedded subtitles do exist with "Use Embedded Subtitles" checked.

**Comment by vitiko98** [comment]
I need the debug log. It's possible that one of the bad subtitles checkers is filtering it.

Please reproduce it and attach the logs.

**Comment by Myrodistus** [comment]
Here's the fresh log after also after deleting The Boys embedded subtitle file/s it found previously so it could grab new ones. Sorry I had to run "Search for wanted Series Subtitles" twice: https://gist.github.com/Myrodistus/2350fbe9ba19d10e6c9e21252bf4f6ef

I also tested this on [v1.2.5-beta.7] earlier today, tried a variety of different settings, it worked for The Boys every time but still not Supernatural.

Hopefully this helps! If anything else is needed I'm periodically checking back here.
